### PR TITLE
New VMware Workstation tutorial

### LINKED ISSUE #462: Document installing VMware Workstation Pro on CL
As suggested in https://github.com/clearlinux/distribution/issues/625, it would be nice for Clear Linux documentation to have a guide on installing VMware Workstation Pro on top of Clear Linux OS (making CL the host OS). 

VMware Workstation Pro is the commercial/paid version of the free VMware Player. 

**Should the new document be a guide, a reference, or a tutorial?**
Getting Started Guide

**Describe or provide examples of similar documents, if possible, from other web sites**
https://wiki.archlinux.org/index.php/VMware

**Additional context**

- Steps to reproduce and issues have been documented in https://github.com/clearlinux/distribution/issues/625.

- This has been made possible recently due to the work done around enabling custom kernel modules. 

- The LTS kernel should be the default recommendation for most likely comparability (similar to the VirtualBox documentation).

### PR DESCRIPTION
**Pull request by krisdale**
Add new VMware Workstation Pro tutorial.

Closes #462

Signed-off-by: Kristal Dale <kristal.dale@intel.com>

### CODE REVIEW

**Comment by krisdale** [comment]
This is WIP and needs some additional info from @puneetse. I will make a note on this PR when it is ready for additional review.

**Comment by krisdale** [comment]
@puneetse Would you review the latest and give your ok (remove the changes requested) if it looks good from your side. That will clear this for final merge. Thanks!

File: source/tutorials/vmware-workstation.rst [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+	  script directories" being given. This can safely be ignored as |CL| uses
+	  :command:`systemd`.
+
+Install VMware kernel modules
+*****************************
+
+Kernel headers are required by VMware Workstation Pro to compile and install the
+necessary kernel modules after installation. Before installing the VMware kernel
+modules, you must first install the :command:`kernel-native-dkms` or
+:command:`kernel-lts-dkms` bundle to provide the tools and Linux headers needed
```
[the remark this answers is not in the document]
Comment (mvincerx) [reply source/tutorials/vmware-workstation.rst]: I realize that you're using an `include` here, which poses a challenge. 

To make it simpler, I recommend removing the final sentence of this para., starting with "Installing..."  Reason: Via the include, you cover the exact same topic. Replace the latter sentence with a one-liner to make the preceding para consistent with the content that appears via the include: "In this tutorial, we install the  `command: kernel-native-dkms` bundle." 

The second sentence of **Install DKMS** Step 2 (via the include) already gives the option of either bundle.

File: source/tutorials/vmware-workstation.rst [hunk 2]
```diff
@@ -0,0 +1,146 @@
+
+.. _vmware-workstation:
+
+VMware Workstation Pro\*
```
[the remark this answers is not in the document]
Comment (krisdale) [reply source/tutorials/vmware-workstation.rst]: Good catch ;)

**Comment by krisdale** [comment]
@mvincerx Thanks for the review - great feedback. I've made the requested changes.

**Comment by mvincerx** [comment]
LGTM. Thanks for all your work on this, @puneetse, @krisdale.

**Pull request merged**